lib/diagnostic.rb

```diff
@@ -13,28 +13,29 @@ def longest(enum)
   # enumerable supplied (i.e. apply the `+` operator)
   # Assume each entry implements a `+` method
   def sum(enum)
-    # Your code goes here
+    enum.reduce(:+)
```

Comment:
Correct, but only in Ruby v2.0 and above. In older versions of Ruby, you need to explicitly add the `&` before the symbol:
`enum.reduce(&:+)`